nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestBinariesObject.java

```diff
@@ -0,0 +1,152 @@
+import org.nuxeo.runtime.management.ManagementFeature;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.TransactionalFeature;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * @since 11.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features(ManagementFeature.class)
+public class TestBinariesObject extends ManagementBaseTest {
+
+    protected static final String FILES_XPATH = "files:files";
+
+    protected static final String GC_BINARIES_PATH = "site/management/binaries/gc/orphaned";
+
+    protected static final String GC_BINARIES_STATUS_PATH = "site/management/binaries/gc/orphaned/inProgress";
+
+    protected static final int DEFAULT_NUMBER_OF_BLOBS = 4;
+
+    @Inject
+    protected TransactionalFeature transactionalFeature;
+
+    @Inject
+    protected CoreSession session;
+
+    @Test
+    public void shouldMarkAndDeleteOrphanedBinaries() throws IOException, InterruptedException {
+        // Create a file document with some blobs
+        DocumentModel document = session.createDocumentModel("/", "myFile", "File");
+        List<Blob> blobsToCreate = createBlobs(DEFAULT_NUMBER_OF_BLOBS);
+
+        List<Map<String, Blob>> files = blobsToCreate.stream().map(b -> {
+            Map<String, Blob> map = new HashMap<>(1);
+            map.put("file", b);
+            return map;
+        }).collect(Collectors.toList());
+
+        document.setPropertyValue("files:files", (Serializable) files);
+
+        session.createDocument(document);
+        transactionalFeature.nextTransaction();
+
+        document = session.getDocument(document.getRef());
+        List<Map<String, Blob>> createdFiles = (List<Map<String, Blob>>) document.getPropertyValue(FILES_XPATH);
+        List<Blob> blobs = createdFiles.stream().flatMap(m -> m.values().stream()).collect(Collectors.toList());
+        long sizeOfBinaries = blobs.stream().mapToLong(Blob::getLength).sum();
+
+        // No binaries orphans, no gc collection
+        makeCallAndVerify(DEFAULT_NUMBER_OF_BLOBS, sizeOfBinaries, 0, 0);
+
+        // Remove the document which will end with orphans blobs
+        session.removeDocument(document.getRef());
+        transactionalFeature.nextTransaction();
+        try {
+            session.getDocument(document.getRef());
+            fail("The document shouldn't exist");
+        } catch (DocumentNotFoundException dnf) {
+            assertNotNull(dnf);
+        }
+
+        // Wait a bit moment, which will allow the gc to collect the unused binaries, when the endpoint will be called
+        // (see BinaryGarbageCollector#stop for more details)
+        Thread.sleep(TIME_RESOLUTION + 1);
+
+        // Now the blobs are orphans, they should be collected
+        makeCallAndVerify(0, 0, DEFAULT_NUMBER_OF_BLOBS, sizeOfBinaries);
+
+        // Now there is no orphans blobs
+        makeCallAndVerify(0, 0, 0, 0);
+
+        // No GC in progress
```

Comment:
```suggestion
        // No gc in progress
```
For consistency with other comments in the file.